CONTRIBUTING.md

```diff
@@ -48,12 +48,12 @@ We strive to follow this [recommendation](https://chris.beams.io/posts/git-commi
 - [Capitalize the subject line](https://chris.beams.io/posts/git-commit/#capitalize).
 - [Do not end the subject line with a period](https://chris.beams.io/posts/git-commit/#end).
 - [Use the imperative mood in the subject line](https://chris.beams.io/posts/git-commit/#imperative).
-- [Wrap the body at 72 characters](https://chris.beams.io/posts/git-commit/#wrap-72).
 - [Use the body to explain what and why vs. how](https://chris.beams.io/posts/git-commit/#why-not-how).
 
 We add this additional rule:
 
 - Do not rely on GitHub issue reference numbers in commit messages, as we have no guarantee the host system and its autolinking will be stable in time. Make sure the context is self-explanatory. If an external reference is given, use its full URL.
+- Do not use [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary) `<type>[optional scope]: <description>`, it will not be allowed, only describe what and why changes were made in one sentence.
```

Comment:
I agree with that, but I feel a rationale why we don't allow it would be welcome 🙂

```suggestion
- Do not use the so-called “[conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)” format (`<type>[optional scope]: <description>`). There is no workflow in place for extracting metadata from such a format and it is thus needlessly cumbersome. 
```

Or maybe don't mention this at all since we already give a reference above?